src/core/kernel/inspect.rs

```diff
@@ -217,6 +216,13 @@ pub(super) fn function_nargs(symbol: &Symbol) -> Result<u32> {
     get_inspector!()?.btf.function_nargs(symbol)
 }
 
+#[allow(dead_code)]
+/// Given an address, gets the name of the nearest symbol, if any.
+pub(crate) fn get_name_from_addr_near(addr: u64) -> Result<String> {
```

Comment:
A possible follow up (maybe in another PR or backlog task) would be to also return the symbol offset.
Of course we can get the disasm and know what line of code but even without that, we could use the offset to  differenciate between two stacks e.g:
```

int foo () +10
    int function_we_are_tracking()
```

vs 
```

int foo () +20
    int function_we_are_tracking()
```

Reply:
Added it with:

pub(crate) fn get_name_offt_from_addr_near(addr: u64) -> Result<(String, u64)>